lib/cluster-providers/generic-cluster-provider.ts

```diff
@@ -3,11 +3,12 @@ import * as ec2 from "aws-cdk-lib/aws-ec2";
 import * as eks from "aws-cdk-lib/aws-eks";
 import { Construct } from "constructs";
 import { ClusterInfo, ClusterProvider } from "../spi";
-import { setPath, valueFromContext, Writeable } from "../utils";
+import * as utils from "../utils";
 import * as constants from './constants';
 import { AutoscalingNodeGroup, ManagedNodeGroup } from "./types";
 import assert = require('assert');
-
+import { StringConstraint, validateConstraints } from '../utils';
```

Comment:
Why are these here? You already imported wildcard from ../utils on line 6.

Reply:
When I get imports I use the quick fix so I don't manually have to type it, and I think it split it up into two for some reason and I did not notice. I should be sure to review my imports in the future if I do that.